libs/common/src/constants/tfm/teams.ts

```diff
@@ -0,0 +1,53 @@
+import { Prettify } from '../..';
+import { TeamId } from '../../types/tfm/team-id';
+
+type TeamRecord = {
+  readonly [Id in TeamId]: Readonly<{
+    id: Id;
+    name: string;
+    email: string;
+  }>;
+};
```

Comment:
This type is slightly better than simply putting `Record<TeamId, Team>` because it ensures two things:
1. That the object includes a value for every `Id` defined in `TeamId`
2. That the value associated with the specific `TeamId` key has the matching `id`

Reply:
See a comment in `libs/common/src/types/tfm/team.ts` related to this